lib/config-schema.js

```diff
@@ -0,0 +1,8 @@
+'use babel'
```

Comment:
@maxbrunsfeld What do you think about this? I know all other Atom packages include the config schema in the main file, but I kinda like separating it and just importing it in main. (It's sort of trivial either way in this case with a config schema so short, but it wouldn't be for packages with more settings.)